README.md

```diff
@@ -216,13 +216,15 @@ To contribute to Covalent, refer to the [Contribution Guidelines](https://github
 
 ## 📝 Release Notes
 
-The latest release of Covalent is now out and available for community use. It comes with new features including the integration of Dask into Covalent as the default executor/compute backend for workflows. As part of this release, we have also updated the CLI to allow users the flexibility to choose the default compute backend between `Dask` and `Local`. The default executor for workflow `electrons` are then set accordingly at runtime. We have also made it simpler for users to parse the logs generated by Covalent by providing a convenient CLI. A summary of the feature releases is provided below
```

Comment:
I don't think you should change the release notes but @santoshkumarradha can confirm that

Reply:
fixed